app/js/components/Artifacts.jsx

```diff
@@ -58,12 +52,13 @@ const Artifacts = ({ artifacts, dispatchDeleteArtifact }) => {
     );
 };
 
+Artifacts.contextTypes = {
```

Comment:
This was from messing around with the redux `subscribe(listener)` where the example I was testing required using `store.getState()` and `store` is only accessible through `this.context`. It might eventually be necessary for showing loading indicators and such, but this was just accidentally left in. I'll cut it out.